examples/v2/project_creation/apis.py

```diff
@@ -19,14 +19,19 @@ def GenerateConfig(context):
 
   project_id = context.properties['project']
   billing = context.properties['billing']
+  concurrent_api_activation = context.properties['concurrent_api_activation']
 
   resources = []
-  for api in context.properties['apis']:
+  for index, api in enumerate(context.properties['apis']):
+    dependsOn = [project_id, billing]
+    # Serialize the activation of all the apis by making api_n depend on api_n-1
+    if (not concurrent_api_activation) and index != 0:
+      dependsOn.append(project_id + '-' + context.properties['apis'][index-1])
```

Comment:
Any use in standardizing the API name now that it's created in two different places? Like, write a function that can create a well-formatted name using this `project_id + '-' + api_name` pattern?